tier2_standards/python_version.md

```diff
@@ -2,16 +2,35 @@
 
 This is one of the tier 2 standards. See full list [on the overview page](README.md).
 
+
 ## Short description
 
+We require that all Python software 'support' python version `3.6` or higher.  We define 'support' in this context such that:
+
+- The Python software is fully operational with no errors when invoked from an interpreter that uses Python `3.6` or higher,
+- Any future contributions to the software must also adhere to the above requirement
+- Any provided environment file(s) in the repository (i.e. `environment.yml`, `requirements.txt`, `setup.py`), must all list a consistent version of Python.
+- Any defined software environments within continuous integration builds (i.e. Jenkins or Travis) must be using the supported Python version(s)
+
 
 ## Importance of this standard
 
+With Python being an free, open-source language, the Python community continues to build and improve the existing language, resulting in periodic releases of new versions of the Python native library as well as many third-party libraries that we commonly use at STScI (e.g. `bokeh`, `astropy`, `numpy`, etc.).  As such, over time the Python software that we write becomes outdated, and the versions of the dependencies that we rely on become unsupported.  This in turn makes it difficult for future developers to maintain existing software and their environments as well as build new features.  For these reasons, we make an effort to keep our software operational with newer Python versions, as does the Python community as a whole, in order to avoid some of these future issues.
 
-## Options for this standard
+Currently, many of the Python libraries we use support Python versions `3.6`, `3.7`, and/or `3.8`.  As such, we are requiring teams to support at least Python version `3.6`.
 
 
 ## How to apply this standard
 
+For software that must be upgraded from Python `2.x` to Python `3.x`, the tool [`2to3`](https://docs.python.org/2/library/2to3.html) can be used to show (and optionally apply) all of the necessary changes.  For a general description of what aspects of the Python `2.x` code must be changed in order to comply with Python `3.x`, see the [official Python documentation on 'updating your code'](https://docs.python.org/3/howto/pyporting.html#update-your-code).
```

Comment:
I just wanted to note that: though it is not necessary for the current lineup of INS JWST software tools since they are all using Python 3, I added this paragraph in case these documents ever apply to newly-adopted tools that are using Python 2.

Reply:
Yes I agree, I think it's a good thing to include to keep this work applicable to a wide set of tools